packages/integration-karma/test/synthetic-shadow/element-api/element-api.spec.js

```diff
@@ -67,11 +67,11 @@ if (!process.env.DISABLE_SYNTHETIC) {
 
         describe('Element.prototype API', () => {
             it('should keep behavior for innerHTML', () => {
-                expect(elementOutsideLWC.innerHTML.length).toBe(455);
+                expect(elementOutsideLWC.innerHTML.length).toBe(27);
```

Comment:
@jodarove I assumed all of these tests were kind of "snapshot" tests, so it was fine to just change these numbers? Also maybe we should look into removing the `ENABLE_ELEMENT_PATCH` flag entirely, unless it's being used somewhere?

Reply:
Now that I think about this again, this change could lead to a regression similar to the `elementsFromPoint` one. With this change, we are indeed closing the gap with native shadow DOM. It's probably something we should discuss as a team how we want to roll this out.

Reply:
I totally agree. Anytime we "fix" something in synthetic shadow, we can break working code. Arguably we should just keep synthetic shadow frozen in amber, and encourage people to migrate to native shadow. (Or put every change behind a flag.) Let's bring this up at DevSync.